common/src/main/java/com/wynntils/core/chat/ChatModel.java

```diff
@@ -243,15 +250,15 @@ private static Component handleChatLine(Component message, String codedMessage,
         return event.getMessage();
     }
 
-    private static void handleNpcDialog(List<String> dialog) {
+    private static void handleNpcDialog(List<String> dialog, boolean autoProgressing) {
```

Comment:
Here, too, I think I prefer `nonBlocking`. (Or, if you want to, inverse the logic and use `blocking`)

Reply:
You resolved this without fixing?

Reply:
Ah, I renamed some other boolean with the wrong terminology.